hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/key/OMKeyCommitRequest.java

```diff
@@ -253,4 +253,33 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
 
     return omClientResponse;
   }
+
+  protected void processResult(CommitKeyRequest commitKeyRequest,
```

Comment:
Seems we forget to reuse processResult in OMKeyCommitRequest.

Reply:
oops, thanks a lot @linyiqun  for the detailed code reviews. I have fixed this in my latest commit.